MAINTAINERS

```diff
@@ -116,6 +116,13 @@ S:	Maintained
 F:	core/arch/arm/plat-imx/
 F:	core/arch/arm/plat-imx/registers
 
+NXP (Freescale) i.MX family - Crypto
```

Comment:
`core/drivers/crypto` has nothing i.MX specific, so perhaps this should be called "Crypto driver interface"?

Reply:
ok